helpers/peer-frontend/contracts/PeerFrontend.sol

```diff
@@ -237,21 +237,21 @@ contract PeerFrontend {
       uint256(keccak256(abi.encodePacked(
```

Comment:
2 lines above this - `consumer`. Can't comment on certain lines of the PR this is very annoying

Reply:
I've found more examples saying `consumer` - maybe do a find for it in this file

Reply:
removed them and redid the comments